README.md

```diff
@@ -12,14 +12,15 @@ back often.
 
 We welcome questions/comments/feedback (and pull requests).
 
-* Mailing List - http://lists.projectcalico.org/listinfo/calico
+* [Announcement Mailing List](http://lists.projectcalico.org/mailman/listinfo/calico-announce_lists.projectcalico.org)

```

Comment:
i think it's nice to put urls like this as references (like we do for IRC) ... define it at the bottom of the file.